engine/schema/src/org/apache/cloudstack/storage/datastore/db/ImageStoreDetailVO.java

```diff
@@ -23,61 +23,61 @@
 import javax.persistence.Id;
 import javax.persistence.Table;
 
-import org.apache.cloudstack.api.InternalIdentity;
+import org.apache.cloudstack.api.ResourceDetail;
 
 @Entity
 @Table(name = "image_store_details")
-public class ImageStoreDetailVO implements InternalIdentity {
+public class ImageStoreDetailVO implements ResourceDetail {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Column(name = "id")
     long id;
 
     @Column(name = "store_id")
-    long storeId;
+    long resourceId;
 
     @Column(name = "name")
     String name;
 
     @Column(name = "value")
     String value;
 
+    @Column(name = "display")
```

Comment:
We added it to make it consistent with `ResourceDetail` as we made `ImageStoreDetailVO` implement `ResourceDetail`, which has the method `isDisplay`.